bitnami/apisix/values.yaml

```diff
@@ -0,0 +1,2818 @@
+  ##
+  hostAliases: []
+  ## @param dataPlane.defaultConfig [string] Apisix apisix configuration (evaluated as a template)
+  ##
+  defaultConfig: |
+    {{- if .Values.dataPlane.metrics.enabled }}
+    plugin_attr:
+      prometheus:
+        export_uri: /apisix/prometheus/metrics
+        metric_prefix: apisix_
+        enable_export_server: true
+        export_addr:
+          ip: 0.0.0.0
+          port: {{ .Values.dataPlane.containerPorts.metrics }}
+    {{- end }}
+    apisix:
+      node_listen: {{ .Values.dataPlane.containerPorts.http }}
+      enable_admin: false
+      {{- if .Values.dataPlane.tls.enabled }}
+      ssl:
+        enable: true
+        listen:
+          - port: {{ .Values.dataPlane.containerPorts.https }}
+            enable_http2: true
+        ssl_trusted_certificate: /bitnami/certs/{{ .Values.dataPlane.tls.certCAFilename }}
+      {{- end }}
+      control:
+        ip: 0.0.0.0
+        port: {{ .Values.dataPlane.containerPorts.control }}
+    nginx_config:
+      error_log: /dev/stderr
+      stream:
+        access_log: /dev/stdout
+      http:
+        access_log: /dev/stdout
+      http_configuration_snippet: |
+        proxy_buffering off;
+    deployment:
+      role: data_plane
+      role_data_plane:
+        config_provider: control_plane
+        {{- if .Values.controlPlane.enabled }}
+        control_plane:
+          host:
+            - {{ ternary "https" "http" .Values.controlPlane.tls.enabled }}://{{ include "apisix.control-plane.fullname" . }}:{{ .Values.controlPlane.service.ports.configServer }}
+          prefix: /apisix
+          timeout: 30
+        {{- end }}
+      {{- if .Values.dataPlane.tls.enabled }}
+      certs:
+        {{- if .Values.dataPlane.tls.enabled }}
+        cert: /bitnami/certs/{{ .Values.dataPlane.tls.certFilename }}
+        cert_key: /bitnami/certs/{{ .Values.dataPlane.tls.certKeyFilename }}
+        {{- if .Values.dataPlane.tls.certCAFilename }}
+        client_ca_cert: /bitnami/certs/{{ .Values.dataPlane.tls.certCAFilename }}
+        {{- end }}
+        {{- end }}
+      {{- end }}
+    discovery:
+      kubernetes:
+        service:
+          # apiserver schema, options [http, https]
+          schema: https #default https
+
+          # apiserver host, options [ipv4, ipv6, domain, environment variable]
+          host: ${KUBERNETES_SERVICE_HOST} #default ${KUBERNETES_SERVICE_HOST}
+
+          # apiserver port, options [port number, environment variable]
+          port: ${KUBERNETES_SERVICE_PORT}  #default ${KUBERNETES_SERVICE_PORT}
+
+        client:
+          # serviceaccount token or token_file
+          token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+
+        default_weight: 50 # weight assigned to each discovered endpoint. default 50, minimum 0
+
+  ## @param dataPlane.existingConfigMap name of a ConfigMap with existing configuration for the apisix
+  ##
+  existingConfigMap: ""
+
```

Comment:
The value `dataPlane.extraConfig` was implemented but is missing in the values.yaml